.circleci/config.yml

```
@@ -34,4 +34,4 @@ workflows:
           name: Ruby << matrix.ruby-version >>
           matrix:
             parameters:
-              ruby-version: ["2.5.9", "2.6.7", "2.7.3", "3.0.1"]
+              ruby-version: ["2.6.7", "2.7.3", "3.0.1", "3.1.0"]
```

this could be, do we need to be specific about the version?
```suggestion
              ruby-version: ["2.6", "2.7", "3.0", "3.1"]
```

---

also why don't we support 2.5 anymore? 😅